src/soroban_rpc.ts

```diff
@@ -25,6 +25,12 @@ export namespace SorobanRpc {
   }
 
   export interface LedgerEntryResult {
+    lastModifiedLedgerSeq?: number;
+    key: xdr.LedgerKey;
+    val: xdr.LedgerEntryData;
+  }
+
+  export interface RawLedgerEntryResult {
```

Comment:
minor, could add js-docs with link to public copy rather than inline comments:
```
/*
* See {@link https://soroban.stellar.org/api/methods/getLedgerEntries#return RPCLedgerEntryResult} 
*/
```

Reply:
nice, done in e7ed703